ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetBlockImporter.java

```diff
@@ -46,20 +45,13 @@ public synchronized boolean importBlock(
         blockValidator.validateAndProcessBlock(
             context, block, headerValidationMode, ommerValidationMode);
 
-    outputs.ifPresent(processingOutputs -> persistState(processingOutputs, block, context));
+    outputs.ifPresent(
```

Comment:
Current behaviour for Mainnet block importer being persisting the state on every block imported. 

Any thoughts on the potential impact this PR might have? ...is the state still persisted on each block imported (perhaps done elsewhere), or is the persist performed less frequently, if so at what cadence?

Reply:
As I read it, it still does persist on every block, here:
https://github.com/hyperledger/besu/blob/9fa9c858ead30d7d1cb82d6435cdaa1c815e5950/ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/AbstractBlockProcessor.java#L165

I'm also going to manually test this for sure since it's such a core change. Just haven't gotten around to it.

Reply:
Sounds like a plan 👍